react/src/theme/components/Badge.ts

```diff
@@ -39,15 +41,17 @@ const variantSubtle = defineStyle((props) => {
   const subtleBgTokenMap: Record<string, string> = {
     main: 'interaction.main-subtle.default',
     sub: 'interaction.sub-subtle.default',
+    info: 'utility.feedback.info-subtle',
     success: 'utility.feedback.success-subtle',
     warning: 'utility.feedback.warning-subtle',
     critical: 'utility.feedback.critical-subtle',
   }
   const subtleColorTokenMap: Record<string, string> = {
-    main: 'utility.feedback.info',
+    main: 'interaction.main.default',
     sub: 'interaction.sub.default',
+    info: 'utility.feedback.info',
     success: 'utility.feedback.success',
-    warning: 'base.content.default',
+    warning: 'yellow.700',
```

Comment:
Why this one not token 🥲

Reply:
@khaleedah why?

Reply:
@dextertanyj  I believe it's because we wanted the content of the badge to be yellow colour to match the warning background better. 

But yellow is abit tricky. If we just put utility.feedback.warning, it would look really off:
<img width="338" alt="image" src="https://github.com/opengovsg/design-system/assets/66247777/10acbbdd-a560-4e7c-8ff7-cc64b55a2206">


And currently we don't have a token name that maps to yellow.700 as it was not widely used in the system. Will check in with Pearly & Khaleedah on this!